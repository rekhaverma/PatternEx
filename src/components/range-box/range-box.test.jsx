import React from 'react';
import moment from 'moment';
import RangeBox from './range-box.component';

/**
 * @status: Complete
 * @sign-off-by: Alex Andries
 */
describe('<RangeBox />', () => {
  let wrapper;
  let props;

  beforeEach(() => {
    props = {
      startDate: moment.utc('2018-01-01'),
      endDate: moment.utc('2018-01-07'),
      currentDate: moment.utc('2018-01-07'),
      updateRange: jest.fn(),
    };
    wrapper = shallow(<RangeBox {...props} />);
  });

  it('should match with snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
