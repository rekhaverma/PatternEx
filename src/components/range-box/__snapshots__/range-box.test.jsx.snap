// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RangeBox /> should match with snapshot 1`] = `
<div
  className="rangeBox"
>
  <span>
    Show: 
  </span>
  <div
    className="rangeBox__input"
    onClick={[Function]}
  >
    <span>
      01-01-2018
    </span>
    <span>
      01-07-2018
    </span>
    <span
      className="icon-Calendar-icon"
      style={
        Object {
          "marginLeft": "auto",
        }
      }
    />
    <span>
      Show: 
    </span>
    <div
      className="rangeBox__input"
      onClick={[Function]}
    >
      <span>
        01-01-2018
      </span>
       -
      <span>
         
        01-07-2018
      </span>
    </div>
    <div
      className="rangeBox__comboBox"
      style={
        Object {
          "display": "none",
        }
      }
    >
      <PredefinedOptions
        activeOption="7days"
        className="rangeBox__predefinedList"
        hasButtons={true}
        onApply={[Function]}
        onCancel={[Function]}
        onClick={[Function]}
        options={Array []}
      />
    </div>
  </div>
</div>
`;
